packages/lib-subject-viewers/src/translations/i18n.js

```diff
@@ -0,0 +1,28 @@
+import i18n from 'i18next'
```

Comment:
Please remove all translation related files and content for a separate PR. Translations will be implemented only if there is platform-level labels specific to this viewer.

Reply:
Done